Sources/Encoder.swift

```diff
@@ -465,4 +465,37 @@ public struct Encoder {
         }
     }
     
+    /**
+     Sets value for provided key path delimited by provided delimiter.
+     
+     - parameter valueToSet:    Value to set
+     - parameter keyPath:       Key path.
+     - parameter withDelimiter: Delimiter for key path.
+     */
+    public static func setValue(dict: inout JSON, value: AnyObject, forKeyPath: String, withDelimiter: String = GlossKeyPathDelimiter) {
```

Comment:
I must disagree - putting this function into the Gloss class would mean that we are extracting a "hey, set this value in this JSON" function to a "master" object, and that function belongs to the Encoding part of the library, so I would leave that there. 

What you think?